cmd/deviceservice/add/add.go

```diff
@@ -1,35 +1,24 @@
 package add
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"html/template"
+
+	"github.com/edgexfoundry-holding/edgex-cli/cmd/deviceservice/update"
```

Comment:
holding drop ... ?

Reply:
good catch up! I haven't noticed it at all. 
It looks to me that moving out of holding haven't happened entirely correctly. 
Will open new issue about this.

Reply:
@jpwhitemn any ideas?